src/v/coproc/tests/utils/wasm_event_generator.h

```diff
@@ -35,9 +35,10 @@ struct cpp_enable_payload {
 /// skipped during serialization, useful for testing failure cases
 struct event {
     std::optional<uint64_t> id;
-    std::optional<ss::sstring> desc;
+    std::optional<bytes> desc;
     std::optional<bytes> script;
     std::optional<bytes> checksum;
+    std::optional<bytes> filename;
```

Comment:
why is this `filename` and not just `fn_name` and `optional<sstring>`

Reply:
this is just test code. Its bytes and not string to emmulate what is performed by rpk. Rpk serializes everything as iobufs in record headers